spec/models/organization_spec.rb

```diff
@@ -6,4 +6,23 @@
   it { should validate_presence_of :login }
   it { should validate_uniqueness_of(:uid) }
   it { should validate_uniqueness_of(:login) }
+
+  describe 'load_organizations' do
+    context 'creates new organization' do
+      let(:whitelisted_organisations) { [ 'test-organization'] }
+      before do
+        stub_const("Organization::WHITELISTED_ORGS", whitelisted_organisations)
+      end
+
+      before do
+        Organization.load_organizations
+      end
+
+      it 'create a new Repository record' do
+        Organization.all.each do |organization|
```

Comment:
Are you expecting several organisations with the same `login` field here? I would just assert that only one organisation is created, and that its' attributes are what you expect.